packages/components/core/styles/common/_tokens.scss

```
@@ -45,3 +45,26 @@ $tokens: meta.module-variables(tokens) !default;
     @include kbq-css-font-variable($component, $property, font-feature-settings, '');
     @include kbq-css-font-variable($component, $property, letter-spacing, '');
 }
+
+@function kbq-sum-css-variables($first, $second) {
+    @return calc(var(--kbq-#{$first}, map.get($tokens, $first)) + var(--kbq-#{$second}, map.get($tokens, $second)));
+}
+
+@function kbq-difference-css-variables($first, $second) {
+    @return calc(var(--kbq-#{$first}, map.get($tokens, $first)) - var(--kbq-#{$second}, map.get($tokens, $second)));
```

Можем применить миксин css-переменных?
```suggestion
    @return calc(kbq-css-variable($first) - kbq-cass-variable($second));
```

---

Предлагаю дать возможность использовать несколько параметров, можно будет переиспользовать в `kbq-css-half-difference`

---

пофиксил